README.md

```diff
@@ -5,6 +5,14 @@
 
 <h1 align="center">P L U T O N O M I C O N</h1>
 
+[![Hercules-ci][Herc badge]][Herc link]
+[![Cachix Cache][Cachix badge]][Cachix link]
+
+[Herc badge]: https://img.shields.io/badge/ci--by--hercules-green.svg
+[Herc link]: https://hercules-ci.com/github/Orbis-Tertius/plutonomicon
+[Cachix badge]: https://img.shields.io/badge/cachix-public_plutonomicon-blue.svg
+[Cachix link]: https://private-plutonomicon.cachix.org
```

Comment:
private-plutonomicon?

Reply:
Oh damn!

thank you @srid 

this means all of my pulls in plutonomicon are equally broken.